README.md

```diff
@@ -96,6 +96,8 @@ Alerts list:
 | nor_distributor_unexpected_errors | Unexpected errors. Check logs for details |
```

Comment:
Last thing - rename alerts in table to names used in alerts.yml

Reply:
Done